components/frontend/src/sharedPropTypes.js

```diff
@@ -180,7 +207,7 @@ export const reportsOverviewPropType = shape({
     comment: string,
     permissions: object,
     title: string,
-    subtitle: string
+    subtitle: string,
```

Comment:
Interesting that sometimes we have trailing slashes in js lists, but most times we don't!

Reply:
Trailing comma's you mean?